### PR TITLE
Add new VK_EXT_multisampled_render_to_single_sampled VUIDs

### PR DESCRIPTION
**Pull request by Tony-LunarG**

### CODE REVIEW

File: layers/drawdispatch.cpp [hunk 1]
```diff
@@ -1019,6 +1060,9 @@ struct DispatchVuidsCmdDrawMeshTasksIndirectCountNV : DrawDispatchVuid {
         dynamic_rendering_06189            = "VUID-vkCmdDrawMeshTasksIndirectCountNV-pDepthAttachment-06189";
```
Comment (Tony-LunarG) [inline layers/drawdispatch.cpp:1060]: Does anyone know why a chunk of these are initialized twice?

Comment (ncesario-lunarg) [reply layers/drawdispatch.cpp:1060]: I'm guessing that is a mistake and the duplicate should be removed.

**Review by ncesario-lunarg — APPROVED** [review]
LGTM!

File: layers/drawdispatch.cpp (hunk 1 above)
Comment (Tony-LunarG) [reply layers/drawdispatch.cpp]: Duplicates removed

**Review by jeremyg-lunarg — APPROVED** [review]
LGTM

**Pull request merged**